src/worker/runtime/DevToolsAdapter.ts

```diff
@@ -37,7 +37,7 @@ export class DevToolsAdapter {
     }
 
     this.devTools!.event({
-      type: 'Runtime.consoleAPICalled',
+      type: 'Runtime.consoleApiCalled',
```

Comment:
I think this one shouldn't be changed. Can you confirm?

Reply:
you're right, nice catch, thanks!